terraform/cloudfront_pub_lambdas.tf

```diff
@@ -0,0 +1,78 @@
+resource "aws_cloudfront_distribution" "lambdas-compiler-explorer-com" {
```

Comment:
Do we really need another cloudfront distribution? What are we distributing?

We already have a handler for `lambda.compiler-explorer.com` in the load balancer, now we have "labmdas" as well as "lambda" ...